src/gui/windows/multiplayer.zig

```diff
@@ -56,11 +56,17 @@ fn join(_: usize) void {
 		ipAddress = "";
 	}
 	if(connection) |_connection| {
+		if (ipAddressEntry.currentString.items.len == 0) {
+			std.log.err("IP address cannot be empty", .{});
+			gui.showNotification("IP address cannot be empty");
+			return;
+		}
 		_connection.world = &main.game.testWorld;
 		main.globalAllocator.free(settings.lastUsedIPAddress);
 		settings.lastUsedIPAddress = main.globalAllocator.dupe(u8, ipAddressEntry.currentString.items);
 		settings.save();
 		main.game.world = &main.game.testWorld;
+		std.log.info("Connecting to world: {s}", .{ipAddressEntry.currentString.items});
```

Comment:
Shouldn't it be "Connecting to server"? Since you are not showing the world name, but the server address.